Sources/FSCameraView.swift

```diff
@@ -190,45 +190,44 @@ final class FSCameraView: UIView, UIGestureRecognizerDelegate {
             return
         }
         
-        DispatchQueue.global(qos: .default).async(execute: { () -> Void in
+        DispatchQueue.global(qos: .default).async(execute: {
```

Comment:
@d-srd maybe go full blown and remove the execute

Reply:
@fajdof done